<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Längenumrechnung</string>
       </attribute>
       <widget class="QComboBox" name="scale">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>86</width>
          <height>25</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1/20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1/24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1/29</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="mm_scale">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>142</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>10</y>
          <width>171</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Länge in inch ''</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="mm">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>142</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Milimeter Mass&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="inch">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>142</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Zoll Mass&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Zoll&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>151</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Länge in Milimeter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Massstab</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Massstäbliche Länge in mm</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ft">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Foot Ft '</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>80</y>
          <width>521</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Warnung: Convertierung von Milimeter nach ' und '' ist fehlerhaft</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Gewichtsumrechnung</string>
       </attribute>
       <widget class="QLineEdit" name="lbs">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Gewicht in LBS</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Gewicht in Kg</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="kg">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="kg_scale">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>118</width>
          <height>3</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>241</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Massstäbliches Gewicht in Kg</string>
        </property>
       </widget>
       <widget class="QComboBox" name="scale_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>86</width>
          <height>25</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1/20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1/24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1/29</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Massstab</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Bildmassstab</string>
       </attribute>
       <widget class="QLineEdit" name="x_orig">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="x_pic">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>X Original</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>10</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>X Bild</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="y_orig">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="y_pic">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Y Original</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>60</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Y Bild</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>30</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Faktor X:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>80</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Faktor Y: </string>
        </property>
       </widget>
       <widget class="QLabel" name="x_factor">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>30</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="y_factor">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>80</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>50</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Einmessen</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="x_messure">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>731</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>X Bild</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="y_messure">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Y Bild</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="x">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>150</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="y">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>200</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>130</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Länge X</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>180</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Länge Y</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>170</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Error Box</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Umrechnen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Löschen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
